### PR TITLE
Pass on the command to redis-cli

### PR DESCRIPTION
**Pull request by kierenevans**
Abstraction fail :D

### CODE REVIEW

**Comment by sroze** [comment]
Haha. By the way, why `bash -c 'container ...'` in the `run` command? You shouldn't need it.

**Comment by kierenevans** [comment]
Sorted 👍 
It was leftover from experimentation with commands, then it got too unwieldy and we moved to functions inside the docker image.

File: redis/3-highly-available/README.md [hunk 1]
```diff
@@ -74,9 +74,8 @@ tasks:
                 cpu: 250m
                 memory: 500Mi
             command:
-              - /bin/bash
-              - -c
-              - 'container run_master_and_sentinel'
+              - container
```
Comment (sroze) [inline redis/3-highly-available/README.md]: you want that on one line tho

Comment (sroze) [reply redis/3-highly-available/README.md]: I mean `container run_master_and_sentinel` on one line

**Pull request merged**